File: Week 1/Mario/mario.c
#include<stdio.h>
#include<cs50.h>

int main (void)

{
    int height;
    
    do
    {
        height = get_int("Height: "); //getting input
    }
    while (height < 1 || height > 8) ;  //input parameters
    
    for (int i = 0; i < height; i++)   //nested loops for spaces and hashes
    {
        int space = (height - 1) - i;
        int hash = (i + 1) ;
        for (int s = 0; s < space; s++) //print spaces
        {
            printf(" ");
        }
        for (int h = 0; h < hash; h++) //print space
        {
            printf("#");
        }
        printf("\n");
    }

}

File: Week 6/Mario/less/mario.py
from cs50 import get_int
space = ' '
sym='#'
while 1:
    height= get_int('height: ')
    if height < 1 or height > 8:
        height = get_int('height: ')
    if height >= 1 or height <= 8:
        break
    height=get_int("Enter height from 1 to 8: ")
for i in range(height):
    print((height-1-i) * space,end="")
    print((i + 1)*sym)